package.json

```diff
@@ -1,16 +1,17 @@
 {
   "name": "vui-alerts",
   "version": "0.1.4",
-  "description": "Mixins and CSS for styling alerts",
+  "description": "Web component alert elements",
+  "private": true,
   "scripts": {
-    "clean": "rimraf *.css && rimraf ./test/*.css",
-    "prebuild": "npm run clean",
-    "build:sass:test": "node-sass --output-style expanded ./test/test.scss ./test/test.css",
-    "build:sass": "node-sass --output-style expanded error.css.scss error.css",
-    "build": "npm run build:sass && npm run build:sass:test",
-    "perceptual-diff": "percy snapshot test/",
-    "pretest": "npm run build",
-    "test": "csslint error.css"
+    "prebuild": "cpy *.html ./bower_components/temp",
+    "build": "vulcanize ./bower_components/temp/alert.html > ./test/perceptual/alert.html.ignore",
+    "postbuild": "rimraf ./bower_components/temp",
+    "preperceptual-diff": "npm run build",
+    "perceptual-diff": "percy snapshot test/perceptual --enable_javascript",
+    "postinstall": "bower install",
+    "screenshot": "phantomjs ./screenshots/build.js",
+    "test": "polylint && wct --skip-plugin sauce"
```

Comment:
I think we can not add the perceptual diff stuff to new repos until Percy gets their web component stuff working. So that removes a lot of the build steps to create temp ignore files and such.